packages/core-compat-api/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@backstage/core-compat-api",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "web-library"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "devDependencies": {
+    "@backstage/cli": "workspace:^",
+    "@backstage/core-app-api": "workspace:^",
+    "@backstage/plugin-puppetdb": "workspace:^",
+    "@backstage/plugin-stackstorm": "workspace:^",
+    "@oriflame/backstage-plugin-score-card": "^0.7.0",
+    "@testing-library/jest-dom": "^5.10.1"
```

Comment:
Why can't this be a `^6.x.y` version? Maybe we just upgraded to 6 during the work on this PR. We should not enforce a `^5` version here :)

Reply:
Hmm, tbh I have no idea why this dep is here to begin with, will fix

Reply:
Oh nvm, now I do 😅, yeah just created before migration